src/components/render-plugin-doc/render-plugin-doc.tsx

```diff
@@ -644,47 +657,86 @@ export class RenderPluginDoc extends React.Component<IProps, IState> {
     );
   }
 
+  private renderLegend(legend) {
+    if (!legend) {
+      return null;
+    }
+
+    if (!Array.isArray(legend)) {
+      legend = [legend];
+    }
+
+    return (
+      <>
+        {': '}
+        <Legend>
+          {legend.map((d, i) => (
+            <>
+              {i ? <br /> : null}
+              {this.applyDocFormatters(d)}
+            </>
+          ))}
+        </Legend>
+      </>
+    );
+  }
+
   private renderChoices(option) {
-    let choices, defaul;
+    let choices,
+      defaultChoice,
+      legends = {};
 
     if (option['type'] === 'bool') {
       choices = ['true', 'false'];
+
       if (option['default'] === true) {
-        defaul = 'true';
+        defaultChoice = 'true';
       } else if (option['default'] === false) {
-        defaul = 'false';
+        defaultChoice = 'false';
       }
     } else {
       choices = option['choices'] || [];
-      defaul = option['default'];
+      defaultChoice = option['default'];
+    }
+
+    if (typeof choices === 'object' && !Array.isArray(choices)) {
+      legends = choices;
+      choices = Object.keys(choices);
     }
 
     return (
-      <React.Fragment>
+      <>
         {choices && Array.isArray(choices) && choices.length !== 0 ? (
           <div>
-            <span className='option-name'>Choices: </span>
+            <span className='option-name'>
+              <Trans>Choices: </Trans>
+            </span>
             <ul>
               {choices.map((c, i) => (
                 <li key={i}>
-                  {c === defaul ? (
-                    <span className='blue'>{c} &nbsp;&larr;</span>
+                  {c === defaultChoice ? (
+                    <span className='blue' title={t`default`}>
+                      <Choice>{c}</Choice> &nbsp;&larr;
+                    </span>
                   ) : (
-                    c
+                    <Choice>{c}</Choice>
                   )}
+                  {this.renderLegend(legends[c])}
                 </li>
               ))}
             </ul>
           </div>
         ) : null}
 
-        {defaul && !choices.includes(defaul) ? (
+        {defaultChoice && !choices.includes(defaultChoice) ? (
```

Comment:
```suggestion
        {defaultChoice !== undefined && !choices.includes(defaultChoice) ? (
```
in case the default is `0`, `false`, or `''`.